.github/workflows/glayout_opamp_sim.yml

```diff
@@ -41,6 +41,8 @@ jobs:
                       pip3 install -r requirements.txt &&\
                       cd ./openfasoc/generators/glayout/tapeout/tapeout_and_RL/ &&\
                       pip3 install prettyprint prettyprinttree gdstk &&\
+                      python3 -m pip uninstall numpy -y &&\
+                      python3 -m pip install numpy==1.23.3 &&\
```

Comment:
can this be implemented in the requirements.txt file instead, pin the version there